repository/repository/routes/api.py

```diff
@@ -28,4 +29,15 @@ def service_status() -> Response:
 def get_eprint_events(identifier: str, version: int) -> Response:
     """Get events for a specific version of an e-print."""
     data, code, headers = controllers.get_eprint_events(identifier, version)
-    return jsonify(data), code, headers
\ No newline at end of file
+    return jsonify(data), code, headers
+
+
+@blueprint.route('/e-print/<arxiv:identifier>v<int:version>/pdf',
+                 methods=['GET'])
+def get_eprint_pdf(identifier: str, version: int) -> Response:
+    """Get PDF for a specific version of an e-print."""
+    pdf, code, headers = controllers.get_eprint_pdf(identifier, version)
+    response = send_file(pdf.content, as_attachment=True,
+                         attachment_filename=pdf.filename,
+                         last_modified=pdf.modified)
+    return response, code, headers
```

Comment:
I probably should have named the return type on the controllers differently than on the routes (both called ``Response``). The former is a three-tuple, the latter is a Flask ``Response`` object. See L20-24 for how I've started to do these, although it may also be prudent to check for duplicate header keys lest we end up with dupes.

E.g.:
```python
for key, value in headers.items():
    if key in response.headers:
        response.headers.remove(key)
    response.headers.add(key, value)
```

Reply:
This should be fixed. I did not do the response header de-duping since that feels like it should be its own pull request.

Reply:
Sounds good